3.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>3</title>
+</head>
+<body>
+    <header>
+        <p><h1>Lorem ipsum</h1></p>
```

Comment:
```suggestion
        <h1>Lorem ipsum</h1>
```
убрать тег p

не ужели читая книгу в параграфе вы можете встретить заголовок